service/src/main/java/tds/config/repositories/ConfigRepository.java

```diff
@@ -2,8 +2,11 @@
 
 import tds.config.ClientSystemFlag;
```

Comment:
Indeed - this has been addressed.  I've separated the time limit-related stuff to its own repository/service/controller.